README.md

```diff
@@ -637,10 +649,17 @@ export default CustomSquare;
 
 ### `styleTagTransform`
 
-Type: `String | Function`
+Type:
+
+```ts
+type styleTagTransform =
+  | string
+  | ((css: string, styleElement: HTMLStyleElement, options: any) => void);
```

Comment:
Why `any`?

Reply:
These options are injected from JS like  
```js
 style.use({
  insertInto: document.body,
  additionalStyles: '.some-element {color: red;}'
});
```
this allows to set any options?

Reply:
Let use `Record<string, any>`, because other types are invalid

Reply:
Done